src/patternfly/components/Sidebar/sidebar.scss

```diff
@@ -88,6 +88,16 @@ $pf-c-sidebar__panel--list--width: ("default", 25, 33, 50, 66, 75, 100);
     }
   }
 
+  &.pf-m-full-height {
+    --pf-c-sidebar__main--AlignItems: stretch;
```

Comment:
This is going to break the sticky behavior by default. I wonder if `stretch` should be the default, and we move sticky to a `[panel/content]-body` element that holds the stuff in those elements.